src/mono/wasm/runtime/pinvoke.h

```diff
@@ -0,0 +1,15 @@
+#ifndef __PINVOKE_H__
```

Comment:
add the license header:

```suggestion
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.

#ifndef __PINVOKE_H__
```